File: Problemas em C ou C plus plus/Problema 1017.cpp
#include <stdio.h>
int main () {
	int tempo_gasto, velocidade_media;
	float litros_gastos, distancia_percorrida;
	
	scanf ("%d", &tempo_gasto);
	scanf ("%d", &velocidade_media);
	
	distancia_percorrida = tempo_gasto*velocidade_media;
	litros_gastos = distancia_percorrida/12;
	
	printf ("%.3f\n", litros_gastos);
	return 0;
}

File: Problemas em Python/Problema 1017.py
tempo_gasto = float(input ("Digite o tempo gasto: "))
velocidade_media = float(input ("Digite a velocidade media: "))

distancia_percorrida = tempo_gasto*velocidade_media
litros_gastos = distancia_percorrida/12;

print ("O valor da quantidade de litros necessarios e: %.3f"%litros_gastos)